### PR TITLE
Purges undertale sounds

### PR DESCRIPTION
**Pull request by PKPenguin321**
Removes all sounds and therefore every Undertale reference except for varying font types for skeletons

Retains skeletony goodness in arcade machines

Closes https://github.com/tgstation/tgstation/pull/25422

### CODE REVIEW

**Comment by optimumtact** [comment]
do not open a pr when an existing pr is open and has not yet been ruled on.

First and only warning.

**Comment by Jordie0608** [comment]
This PR is different in function, and should not be treated as an accessory, to #25422. @optimumtact I'm going to reopen this unless you have a better reason for it to be closed.

**Comment by PKPenguin321** [comment]
I even tested this to confirm that it works

**Comment by Iamgoofball** [comment]
@Jordie0608 i meant to remove the files in my version

**Comment by ShadowDeath6** [comment]
a better reference is to that rattle me bones commercial

**Comment by PKPenguin321** [comment]
i considered it

**Comment by Iamgoofball** [comment]
this has been merged into my PR, close this please

**Comment by PKPenguin321** [comment]
No, this one repurposes the skeleton toys so they're not undertale references anymore without tearing them out of the game completely and isn't a WIP

**Comment by Niknakflak** [comment]
I tried to do this once, and it was closed

**Comment by Thunder12345** [comment]
>Chattering bone tounge that doesn't even chatter

File: code/modules/surgery/organs/organ_internal.dm [hunk 1]
```diff
@@ -653,7 +645,6 @@
 
 /obj/item/organ/tongue/bone/chatter
```
Comment (Jordie0608) [inline code/modules/surgery/organs/organ_internal.dm]: Just remove the whole type as it's now function-less.

File: code/modules/surgery/organs/organ_internal.dm [hunk 2]
```diff
@@ -627,21 +627,13 @@
 	attack_verb = list("bitten", "chattered", "chomped", "enamelled", "boned")
 	taste_sensitivity = 101 // skeletons cannot taste anything
 
-	var/chattering = FALSE
 	var/phomeme_type = "sans"
 	var/list/phomeme_types = list("sans", "papyrus")
```
Comment (ShadowDeath6) [inline code/modules/surgery/organs/organ_internal.dm]: Is this used for anything other than new() ?

File: code/modules/surgery/organs/organ_internal.dm (hunk 1 above)
Comment (PKPenguin321) [inline code/modules/surgery/organs/organ_internal.dm]: I left it since I'm worried it's used in some maps

Comment (PKPenguin321) [inline code/modules/surgery/organs/organ_internal.dm]: I left it since I'm worried it's used in some maps but I'll have a look later

**Comment by ExcessiveUseOfCobblestone** [comment]
Purpose of the PR?

**Comment by Cyberboss** [comment]
Goof salt

**Comment by ShadowDeath6** [comment]
Rip pr

**Comment by PKPenguin321** [comment]
You merged the wrong one reeeee

**Comment by calzilla1** [comment]
Should just say "raddle me bones!"

**Comment by PKPenguin321** [comment]
After the freeze I'll try to readd just the skeleton toys

**Pull request closed without merging**